pkgs/development/tools/analysis/qcachegrind/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, bash, cmake, qt57, graphviz,  qmakeHook, makeQtWrapper, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "kcachegrind-${version}";
```

Comment:
Hmm, then I would suggest changing the name here to qcachegrind and change the source URL to `http://download.kde.org/stable/applications/${version}/src/kcachegrind-${version}.tar.xz`.

The package name should reflect what you actually get when installing the package and doing `nix-env -i kcachegrind` and ending up with qcachegrind may be confusing.